opennebula/shared_schemas.go

```diff
@@ -716,5 +716,28 @@ func flattenUserTemplate(d *schema.ResourceData, vmTemplate *dynamic.Template) e
 		}
 	}
 
+	memory, _ := vmTemplate.GetInt("MEMORY")
```

Comment:
I'll take care to merge #236 just before this PR, it already contains basic reading code for CPU, memory, VCPU, could your remove all code of the shared_schemas.go file from this PR please ? (I already asked but maybe that I didn't explained well)